analytics/views/installation_activity.py

```diff
@@ -250,28 +251,34 @@ def realm_summary_table() -> str:
         total_bot_count += int(row["bot_count"])
         total_wau_count += int(row["wau_count"])
 
-    total_row = dict(
-        string_id="Total",
-        plan_type_string="",
-        org_type_string="",
-        effective_rate="",
-        arr=total_arr,
-        realm_url="",
-        stats_link="",
-        support_link="",
-        date_created_day="",
-        dau_count=total_dau_count,
-        user_profile_count=total_user_profile_count,
-        bot_count=total_bot_count,
-        wau_count=total_wau_count,
-    )
-
-    rows.insert(0, total_row)
+    total_row = [
+        "Total",
```

Comment:
This isn't great, but refactoring how this whole chart and page is rendered isn't a top priority. At the same time, having the total row with this new formatting and the ARR column formatted seem worth doing in the short term.

Reply:
Yeah, we can live with this.